BaseFilter/sections/adservers.txt

```diff
@@ -1105,3 +1105,257 @@
 !
 ! https://github.com/AdguardTeam/AdGuardHome/issues/1046
 ||samsungtvads.com^
+!
```

Comment:
I'd wrap it by Comment Anchors tags

* Beginning
```
! NOTE: CNAME disguised advertising networks
! SECTION: CNAME disguised advertising networks
```

* End
```
! NOTE: CNAME disguised advertising networks end ⬆️
! !SECTION: CNAME disguised advertising networks
```

And same for tracking.